gin_test.go

```diff
@@ -8,11 +8,60 @@ import (
 	"reflect"
 	"testing"
 
+	"net/http"
+
```

Comment:
Sorry for the miss inspection, this was auto import package and format style by the vscode.